common/src/main/java/com/revenuecat/purchases/common/BillingAbstract.kt

```diff
@@ -79,7 +79,8 @@ abstract class BillingAbstract {
         appUserID: String,
         purchasingData: PurchasingData,
         replaceProductInfo: ReplaceProductInfo?,
-        presentedOfferingIdentifier: String?
+        presentedOfferingIdentifier: String?,
+        isPersonalizedPrice: Boolean? = null
```

Comment:
it would be a lot more ideal for us to be able to pass the `PurchaseParams` to this method, considering we now have both the `replaceProductInfo` and `isPersonalizedPrice` that are google-specific... but unfortunately, almost all of the ProductChange flow is in Purchases, and it would be a pretty major refactor to pull it into BillingWrapper.

definitely something to think about in the future though.

Reply:
Could `isPersonalizedPrice` be added into `PurchasingData` since we have a `GooglePurchasingData` and `AmazonPurchasingData` 🤔

Reply:
discussed on slack - doesn't feel right since `GooglePurchasingData` is in a storeproduct, and ispersonalizedprice is purchase-specific